pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -129,41 +225,30 @@
                 <% } %>
               <tr>
                 <% if (assessment.group_work) { %>
-                  <td><%= row.group_name %></td>
-                  <td class="text-center" style ="word-break:break-all;"><small><%= row.uid_list %></small></td>
+                  <td class="align-middle"><%= row.group_name %></td>
+                  <td class="text-center align-middle" style ="word-break:break-all;"><small><%= row.uid_list %></small></td>
                 <% } else { %>
-                <td><%= row.uid %></td>
-                <td><%= row.name %></td>
-                <td class="text-center"><%= row.role %></td>
+                <td class="align-middle"><%= row.uid %></td>
+                <td class="align-middle"><%= row.name %></td>
+                <td class="text-center align-middle"><%= row.role %></td>
                 <% } %>
-                <td class="text-center"><%= row.number %></td>
+                <td class="text-center align-middle"><%= row.number %></td>
                 <td class="text-center align-middle"><%- include('../partials/scorebar', {score: row.score_perc}); %></td>
-                <td class="text-center"><%= row.date_formatted %></td>
-                <td class="text-center" data-text="<%= row.duration_secs %>"><%= row.duration %></td>
-                <td class="text-center"><%= row.time_remaining %></td>
-                <td class="text-center">
+                <td class="text-center align-middle"><%= row.date_formatted %></td>
+                <td class="text-center align-middle" data-text="<%= row.duration_secs %>"><%= row.duration %></td>
+                <td class="text-center align-middle">
+                  <%= row.time_remaining %>
+                  <% if (authz_data.has_instructor_edit) { %>
+                  <a class="btn btn-xs time-limit-edit-button" data-row='<%= JSON.stringify(row) %>' >
```

Comment:
Nit: single quotes aren't necessary since EJS will escape the JSON string for you.

```suggestion
                  <a class="btn btn-xs time-limit-edit-button" data-row="<%= JSON.stringify(row) %>">
```